src/webui/api/transfercontroller.cpp

```diff
@@ -71,6 +75,8 @@ void TransferController::infoAction()
     dict[KEY_TRANSFER_UPDATA] = static_cast<qint64>(sessionStatus.totalPayloadUpload);
     dict[KEY_TRANSFER_DLRATELIMIT] = BitTorrent::Session::instance()->downloadSpeedLimit();
     dict[KEY_TRANSFER_UPRATELIMIT] = BitTorrent::Session::instance()->uploadSpeedLimit();
+    dict[KEY_TRANSFER_LAST_EXTERNAL_ADDRESS_V4] = BitTorrent::Session::instance()->lastExternalIPv4Address();
```

Comment:
BTW I would suggest assigning `const auto *btSession = BitTorrent::Session::instance();` at the top of this function to avoid repetitive calls.